RecoLocalCalo/HcalRecProducers/python/HcalHitReconstructor_hbhe_cfi.py

```diff
@@ -31,7 +31,9 @@
     setTimingShapedCutsFlags  = cms.bool(True),
     setTimingTrustFlags       = cms.bool(False), # timing flags currently only implemented for HF
     setPulseShapeFlags        = cms.bool(True),
-    setNegativeFlags          = cms.bool(True), # only in HBHE
```

Comment:
I suggest to be more explicit and to set it to False here instead of commenting out